src/containerapp/azext_containerapp/_params.py

```diff
@@ -94,6 +94,7 @@ def load_arguments(self, _):
         c.argument('registry_pass', validator=validate_registry_pass, options_list=['--registry-password'], help="The password to log in to container registry. If stored as a secret, value must start with \'secretref:\' followed by the secret name.")
         c.argument('registry_user', validator=validate_registry_user, options_list=['--registry-username'], help="The username to log in to container registry.")
         c.argument('secrets', nargs='*', options_list=['--secrets', '-s'], help="A list of secret(s) for the container app. Space-separated values in 'key=value' format.")
+        c.argument('registry_identity', help="A Managed Identity to authenticate with the registry server instead of username/password. Use a resource ID or 'system' for user-defined and system-defined identities, respectively. The registry must be an ACR. If possible, an 'acrpull' role assignemnt will be created for the identity automatically.")
```

Comment:
Could you follow this guideline [managed_identity_command_guideline.md](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design managed identity related parameters?

Reply:
I don't think these guidelines are relevant to this use case. This parameter isn't for assigning a managed identity to the containerapp (that's a different set of parameters), it's for using a managed identity to pull images from an ACR (instead of username/password).

Reply:
Got it, thanks~